src/modules/productTile/productPrice/productPrice.js

```diff
@@ -1,40 +1,46 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { P, formatPrice } from 'SRC'
+import { P, formatPrice, theme } from 'SRC'
 
+const Text = styled(P)`
+  font-weight: 500;
+  font-size: 14px;
+`;
 
-const BaseProductPrice = ({ colorway, className }) => {
-  const formatSalePrice = (price) => {
-    const decimalPlaces = parseInt(price, 10) === parseFloat(price) ? 0 : 2
-    return formatPrice(price, "$", decimalPlaces)
-  }
+const formatSalePrice = (price) => {
+  const decimalPlaces = parseInt(price, 10) === parseFloat(price) ? 0 : 2
+  return formatPrice(price, "$", decimalPlaces)
+}
 
+const BaseProductPrice = ({ colorway, className }) => {
   const originalPrice = colorway.skus[0].original_price
   const price = colorway.skus[0].price
   const onSale = originalPrice && originalPrice !== 0 && price < originalPrice
+  const promoPrice = parseFloat(price) * 0.8
```

Comment:
Should we have the discount as a prop passed into the product tile?

Reply:
We may also want to have the number of items the customer needs to add to be eligible for the promo passed in as a prop